fixed typos (#10)
appwrite/demo-almost-netflix-for-flutter: README.md

### `lib/extensions`

> We use the `lib/extensions` folder to extend the Dart language with helpers for convenience methods.

Reviewer: ```suggestion
We use the `lib/extensions` folder to extend the Dart language with convenient helper methods.
```

### `lib/providers`

Our `lib/providers` folder is where we create our `ChangeNotifiers` that allow observation and access across our app.

> For more information about the Provider library which we used can be found in the [documentation](https://pub.dev/packages/provider)

Reviewer: ```suggestion
More information about the Provider library we used can be found in their [documentation](https://pub.dev/packages/provider)
```
